fixed grammar issue on about page

jossters opened the pull request:
# Description
<img width="635" alt="Screen Shot 2021-08-17 at 9 33 32 PM" src="https://user-images.githubusercontent.com/72762865/129857615-02d8649a-a77c-4c41-bd4e-d6f98da9163a.png">
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue

## Change Status

- [x] Complete, tested, ready to review and merge


# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts
- [ ]

jossters committed:
fixed grammar issue on about page

jossters committed:
Merge branch 'main' of https://github.com/Lambda-School-Labs/human-rights-first-police-fe-a into feature/AboutPageGrammar
hello

JAaron93 reviewed (approved):
I like what you did here. 

- Clear and concise language
- Updated the length of time we've been developing this from six to seven months
- Capitalized Twitter 
- Removed mentions of Reddit
- Corrected typos

Only nitpicky thing I'd change is the last sentence:
', that have ever contributed to this project.'

Otherwise it's pretty much perfect. Might add some more details next week about the bot once our team has actually completed it.

alvin-joseph commented on src/components/about/About.js:
```diff
@@ -114,31 +113,30 @@ const About = () => {
+              nearly eight months of work by Lambda School students. Below are
+              all the students, and their GitHub profiles, that have ever worked
               on this website.
             </p>
```
Nice work updating this text! It definitely needed it! I agree with Joshua about the last sentence though. I would maybe change it to "Below are all of the Lambda School students that have worked on this website, including their GitHub profiles."

devin-mitchell replied:
I agree with Alvin and Joshua.  Little easier to read/understand.

jossters replied:
How about 'all the' instead of 'all of the'.

alvin-joseph replied:
Yeah that sounds better actually

RyanFikejs commented on src/components/about/About.js:
```diff
@@ -12,10 +12,10 @@ const About = () => {
       <div className="about-top-container">
         <div className="data-description-container">
-          <h1>How Our Data Is Collected</h1>
+          <h1>How We Collected Data</h1>
```
Avoiding the whole "data is" vs. "data are" debacle. Clever wording!

RyanFikejs commented on src/components/about/About.js:
```diff
@@ -37,10 +36,10 @@ const About = () => {
                 PB2020/police-brutality GitHub Repository
               </a>
               {'. '}
-              This repository accumulates and contextualizes crowdsourced
```
I'm re-reading this thing multiple times to be sure: I think this was actually ok as "contextualizes" vs. "contextualize".

alvin-joseph replied:
Yeah I agree. I kept second guessing myself when reading that lol

RyanFikejs commented on src/components/about/About.js:
```diff
@@ -37,10 +36,10 @@ const About = () => {
               resource is Twitter. We continually search Twitter for evidence of
-              police use of force with a twitter bot using a technique called
+              police use of force with a Twitter bot using a technique called
               Natural Language Processing.
```
Consider adding that the specific NLP model used here is a tailored version of BERT. I also suggest including a link to some information about BERT: https://en.wikipedia.org/wiki/BERT_(language_model)

jossters replied:
I agree, ill include BERT.

RyanFikejs commented on src/components/about/About.js:
```diff
@@ -49,23 +48,23 @@ const About = () => {
-              use-of-force occurred and were reported but do not make it into
-              our system). To combat this, each incident report needs to be
-              approved by the Blue Witness Administrative Team.
+              We generated a model within our database to categorize each
```
Suggesting we change to:
"Our BERT model categorizes each incident report we receive."

RyanFikejs commented on src/components/about/About.js:
```diff
@@ -49,23 +48,23 @@ const About = () => {
-              use of force incident, then it's marked to be reviewed. After an
+              If a gathered incident report meets the criteria of valid police
+              use of force, the incident will be marked and reviewed. After an
               identified incident report has been approved, it is added to the
```
Technically the incident report exists in our database already and its status is updated to 'approved' or 'rejected'. By keeping both in the database we are able to be sure that our twitter scraper doesn't pull in a duplicated instance in our database.

jossters replied:
I also used this in the 'confirm' section.

RyanFikejs replied:
Good thinking!

RyanFikejs commented on src/components/about/About.js:
```diff
@@ -96,7 +95,7 @@ const About = () => {
               for human rights, so we press the U.S. government and private
               companies to respect human rights and the rule of law. When they
-              fail, we step in to demand reform, accountability and justice.
+              fail, we step in to demand reform, accountability, and justice.
```
Thank you for including an Oxford comma. I know that they're considered "optional" in conventional, American English, but I still find them very useful.

devin-mitchell commented on src/components/about/About.js:
```diff
@@ -114,31 +113,30 @@ const About = () => {
-              The goal of the Twitterbot on the Blue Witness project is to
-              adequately scrape twitter for reports of incidents of police
+              The goal of the Twitter bot on the Blue Witness project is to
+              adequately scrape Twitter for reports of incidents of police
```
very nit picky, but could maybe change this sentence to something like "...reports regarding incidents of police..." just to have a bit more flow?  Just a thought, feel free to do whatever you want.

jossters replied:
Yeah, I agree it sounds better thanks.

devin-mitchell reviewed (approved):
Looks great in my opinion.  Left one very small note to possibly consider.

RyanFikejs commented on src/components/about/About.js:
```diff
@@ -114,31 +113,30 @@ const About = () => {
             <p>
-              The goal of the Twitterbot on the Blue Witness project is to
-              adequately scrape twitter for reports of incidents of police
+              The goal of the Twitter bot on the Blue Witness project is to
```
We need to establish a differentiation here between our Twitter scraper and our Twitter bot. The scraper is responsible for gathering tweets with content specific criteria including (but not limited to) key words such "police" and "misconduct" and their synonyms. It does this multiple times a day (I think every 4 hours) and checks the incoming data against data in our database to prevent duplicates being entered into the database and then passes them to the BERT model to be classified and await administrator approval.

The Twitter bot is a tool that we are currently building that is able to reach out to Twitter users to confirm or supply missing information. It is also triggered by the Administration team and new data gathered awaits approval before being marked as an 'approved' incident report.

RyanFikejs replied:
(I think that the previous Labs cohorts had not fully made this distinction clear in the about section because they had not yet reached that stage of development.)

jossters replied:
I'm using this to update the section 'Our Twitter Scrapper/Bot'

RyanFikejs commented on src/components/about/About.js:
```diff
@@ -114,31 +113,30 @@ const About = () => {
-              nearly seven months of work by Lambda School students. Below are
-              all the students, and their Github profiles, that have ever worked
+              nearly eight months of work by Lambda School students. Below are
+              all the students, and their GitHub profiles, that have ever worked
```
Suggesting:
"Below are the students who have worked on this project with links to their Github profiles included.

RyanFikejs commented:
Johnny, thank you for tackling this very necessary sweep of the about section. Don't let the number of my comments dissuade you, you're doing good work, we just have a few things to address before merging. If you have any questions and I don't appear to be responsive here, please feel free to DM me on slack for more clarification. (I will try to check here first.)

devbrunopaula reviewed (commented):
Great changes 👏

jossters committed:
made changes to about page

RyanFikejs commented on src/components/about/About.js:
```diff
@@ -113,31 +121,34 @@ const About = () => {
-              incident map as well as the data available for download to
-              researchers, journalists, students, and activists. If you'd like
-              to know more, feel free to reach out to HRF on our contact page.
+              The Twitter scrapper on the Blue Witness project is responsible
```
I was going to say to just change "scrapper" to "scraper" and I think we're looking good, but I went ahead and committed that change on my own.

RyanFikejs committed:
Update About.js

I just changed "scrapper" to "scraper" so we don't have to wait until you're back on to merge your pull request.

RyanFikejs reviewed (approved):
Thanks for taking the time to go over the changes again Johnny! I feel like this work is sometimes thankless and never ending, so thanks for getting it done!
